ReactCommon/cxxreact/Value.h

```diff
@@ -13,6 +13,7 @@
 #include <JavaScriptCore/JSValueRef.h>
 
 #include <folly/dynamic.h>
+#include <jni/LocalString.h>
```

Comment:
Would it make sense to throw some `#ifdef __ANDROID__` pragmas around it and the `str` method? At least that way iOS can't inadvertently depend on it.